docs/examples/examples.md

```diff
@@ -82,6 +82,15 @@ Please note that these examples, although similar, are distinct from the ones us
 * [readme](https://github.com/earthly/earthly/tree/main/examples/readme) - some sample code we used in our README.
 * [tests](https://github.com/earthly/earthly/tree/main/tests) - a suite of tests Earthly uses to ensure that its features are working correctly.
 
+### Larger Examples And Community Examples
+
+* [Earthly, Rust, GoLang, NodeJS and GitHub Actions Example](https://github.com/earthly/earthly-vs-gha)
+* [Cloud Services In GoLang](https://github.com/earthly/cloud_services_example)
```

Comment:
Nit: We generally use kebab-case for repo names.

Reply:
Fixed